radicle-cli/Cargo.toml

```diff
@@ -7,9 +7,11 @@ edition = "2021"
 
 [dependencies]
 anyhow = { version = "1" }
+chrono = { version = "0.4", default-features = false, features = ["clock", "std"] }
 console = { version = "0.15" }
 dialoguer = { version = "0.10.0" }
 indicatif = { version = "0.16.2" }
+json-color = "0.7"
```

Comment:
Can you use the expanded syntax for consistency?

Reply:
hm... I think you're talking about `chrono`, review is showing `json-color`, look at `log`, it also has features. want me to expand that as well?

Reply:
no I mean `json-color = { version = "0.7" }`

Reply:
ok, [fixed](https://github.com/radicle-dev/heartwood/compare/86f46c16b9ea02c83094179ff3338631d94a608f..fb80b62989b5019509654551bde0d2dd6ddf8ba2#diff-7522ef468e09bf7246e59e2a6e819722d9213f7ad490c05e0a99b497ce4fcb7cR10-R16)